withdrawn-packages.txt

```diff
@@ -19,3 +19,7 @@ vitess-20.0.2-r5.apk
 vitess-20-compat-20.0.2-r0.apk
 vitess-20-20.0.2-r0.apk
 vitess-20-binaries-20.0.2-r0.apk
+management-api-for-apache-cassandra-0.1.82-r0.apk
+management-api-for-apache-cassandra-compat-0.1.82-r0.apk
+management-api-for-apache-cassandra-4.1-compat-0.1.87-r0.apk
+management-api-for-apache-cassandra-4.1-0.1.87-r0.apk
```

Comment:
We wouldn't pull the packages, just delete the package definition

Reply:
Thanks @EyeCantCU ,  I think that's what you meant what I just pushed by deleting the changes in withdrawn-packages.txt?

Reply:
Perfect!